tests/unit/withOnyxTest.js

```diff
@@ -38,12 +38,12 @@ describe('withOnyx', () => {
             })
             .then(() => {
                 const textComponent = result.getByText('test1');
-                expect(textComponent).toBeTruthy();
-                expect(result).toHaveBeenCalledTimes(999);
+                expect(textComponent).not.toBeNull();
```

Comment:
And then started to fail after it was fixed. I guess `toHaveBeenCalledTimes(999)` was some leftover...